apps/src/templates/projects/projectsModule.js

```diff
@@ -15,7 +21,52 @@ export function selectedGallery(state, action) {
   }
 }
 
+export function projectLists(state, action) {
+  // A map from project type to array of projects, e.g.:
+  //   {
+  //     applab: [{...}, {...}, {...}]
+  //     gamelab: [{...}, {...}, {...}]
+  //     playlab: [{...}, {...}, {...}]
+  //     artist: [{...}, {...}, {...}]
+  //   }
+  state = state || {};
+  switch (action.type) {
+    case SET_PROJECT_LISTS:
+      return action.projectLists;
+    case ADD_OLDER_PROJECTS: {
+      // Append the incoming list of older projects to the existing list,
+      // removing duplicates.
+      const {projects, projectType} = action;
+      state = {...state};
+      state[projectType] = _.unionBy(state[projectType], projects, 'channel');
+      return state;
```

Comment:
I'm going to propose punting on Immutable for now. It seems like one of the possible outcomes of rethinking how the projects redux code should be laid out might be combining all of the reducers into a single reducer, which would make this slightly easier in the future.